src/js/components/inspectors/Property.jsx

```diff
@@ -106,7 +103,6 @@ var Property = React.createClass({
             <div>
               <input type="color" value={!disabled && value} disabled={disabled}
                 onChange={this.handleChange} />
-              <ContentEditable {...props} />
```

Comment:
My bad, I couldn't see them in the UI or didn't know how to use them, or because I was gravitating towards the color picker... since it didn't visibly change the UI to remove them and it made my ContentEditable vs SignalValue refactor easier to remove them, I did so. The use-case you describe makes sense and I'd be in favor of reinstating them.